packages/generator/src/generator.js

```diff
@@ -47,6 +54,8 @@ export default class Generator {
     // Wait for nuxt be ready
     await this.nuxt.ready()
 
+    process.EXPORT_HASH = createTimeHash()
```

Comment:
This is making side-effects. IE: Two generator instances in the same process will conflict.

Reply:
Could you elaborate on that?

Reply:
We are using a shred reference inside class to store state. This is not a good practice. Please review other comments regarding storing hash in builder instance (and inheriting by generator / generate one if no buolder provided)